### ISSUE TITLE
Feature Request: Multi currencies on Expenses module

### ISSUE THREAD

**Issue by asolslk** [Dolibarr/dolibarr#14348]
# Instructions
*This is a template to help you report good issues. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *replace the bracket enclosed texts with meaningful information*
- *remove any unused sub-section*

# Feature Request
[*Short description*]
Would like to request to have the multicurrency for the expenses submission.

## Use case
[*Verbose description*]

## Suggested implementation
[*Verbose description*]

## Suggested steps
[*List of tasks to achieve goal*]

**Comment by edupulpillo** [comment]
Without the Multicurrency support, unfortunately the expenses module is not useful in a lot of situations. 
Hoping this feature will be implemented soon.

**Comment by henrynopo** [comment]
same request here.

**Comment by edupulpillo** [comment]
Nobody else thinks is a feature that need to be implemented to make dolibarr really multicurrency ? :-)

**Comment by soehlrich** [comment]
Yes, would really like this feature, as we have employees overseas wanting to use expense accounts. They are also paid out of foreign curreny accounts.
Does any know if this will happen ?

**Comment by milenmk** [comment]
For now, the only option is an external module you can find on [Dolistore](https://www.dolistore.com)

**Comment by edupulpillo** [comment]
I did not see any module with multicurrency support in expenses management. What is the name of the module that you propose ?

**Comment by milenmk** [comment]
Search for "Expensereport Multi Currency".

**Comment by edupulpillo** [comment]
Nobody think that multicurrency support is necessary for tghe expenses module?

**Comment by soehlrich** [comment]
Yes, it would be a good idea






Sent from my Galaxy







-------- Original message --------
From: Eduardo Pulpillo  
Date: 8/12/23 7:40 pm (GMT+12:00) 
To: Dolibarr/dolibarr  
Cc: soehlrich , Comment  
Subject: Re: [Dolibarr/dolibarr] Feature Request: Multi currencies on
Expenses module (#14348)

Yes, it would be a good idea






Sent from my Galaxy







-------- Original message --------
From: Eduardo Pulpillo  
Date: 8/12/23 7:40 pm (GMT+12:00) 
To: Dolibarr/dolibarr  
Cc: soehlrich , Comment  
Subject: Re: [Dolibarr/dolibarr] Feature Request: Multi currencies on
Expenses module (#14348) 








Nobody think that multicurrency support is necessary for tghe expenses
module?



—
Reply to this email directly, view it on GitHub [1], or unsubscribe
[2].
You are receiving this because you commented.Message ID: 



Links:
------
[1]
https://github.com/Dolibarr/dolibarr/issues/14348#issuecomment-1846625890
[2]

**Comment by milenmk** [comment]
It's not a must. That's why I've created a third party module available on Dolistore. Just search  for "Expensereport Multi Currency".

**Comment by edupulpillo** [comment]
Well, is supposed that Dolibarr aims to be a Multicurrency solution, without too much modules necessary for this small things. do not you think so? Even your module is very welcome, is better to have the multicurrency feature integrated in the main Dolibarr standard verion, i think

**Comment by josett225** [comment]
I agree that multicurrency is required in the expense module too

**Comment by josett225** [comment]
Still required

**Comment by edupulpillo** [comment]
Still required
